### ISSUE TITLE
Event subscription cannot be renewed (events_asyncio)

### ISSUE THREAD

**Issue by cyrilwanner** [SoCo/SoCo#822]
## Environment

SoCo version: `0.22.1`
SoCo events module: `events_asyncio`
Python version: `3.8`, `3.7`

## Error

When trying to renew an event subscription (either with `auto_renew=True` or manually calling `await renew()`), the following error is thrown (`coroutine 'Subscription._request.<locals>._async_make_request' was never awaited`) and the subscription is not renewed:

```
.../python3.8/asyncio/base_events.py:1860: RuntimeWarning: coroutine 'Subscription._request.<locals>._async_make_request' was never awaited
  handle = None  # Needed to break cycles when an exception occurs.
Object allocated at (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/soco/events_asyncio.py", lineno 491
    return _async_make_request()
```

## Code to reproduce this issue

Use the [example from the docs page](http://docs.python-soco.com/en/v0.22.0/api/soco.events.html#module-soco.events_asyncio) and modify the main function like this:

```diff
async def main():
    # pick a device at random and use it to get
    # the group coordinator
    loop = asyncio.get_event_loop()
    device = await loop.run_in_executor(None, _get_device)
    sub = await device.renderingControl.subscribe()
    sub2 = await device.avTransport.subscribe()
    sub.callback = print_event
    sub2.callback = print_event

    async def before_shutdown():
        await sub.unsubscribe()
        await sub2.unsubscribe()
        await events_asyncio.event_listener.async_stop()

+    await asyncio.sleep(10)
+    print('Renewing subscription..')
+    await sub.renew()

    await asyncio.sleep(100)
    await before_shutdown()
```

**Comment by pwt** [comment]
Hi @bdraco : Do you feel like taking a look at this one?

**Comment by bdraco** [comment]
I wasn't able to replicate with pysonos, but there have been some downstream fixes so I'll try directly soon.

I'll open PRs to get the fixes into SoCo later today and then retest directly.

**Comment by bdraco** [comment]
Fix for the session race is here https://github.com/SoCo/SoCo/pull/823. black version seems to have changed and its failing on existing files

Working on replication now

**Comment by bdraco** [comment]
@cyrilwanner Can you give https://github.com/SoCo/SoCo/pull/824 a shot?

**Comment by pwt** [comment]
> Fix for the session race is here #823. black version seems to have changed and its failing on existing files.

As noted in the PR, #825 has the changes to keep Black happy.

**Comment by cyrilwanner** [comment]
> @cyrilwanner Can you give #824 a shot?

I can confirm that #824 fixes this bug, both for the manual renewal, but also for `auto_renew=True` which had the same error.
Thank you so much for resolving this issue so fast!

**Comment by pwt** [comment]
> I can confirm that #824 fixes this bug, both for the manual renewal, but also for `auto_renew=True` which had the same error.

Great -- thanks for the quick confirmation. Once I have a final PR, I'll cut a v0.22.2 release containing the fix.

**Comment by bdraco** [comment]
Rebased, good to go now 👍

**Comment by pwt** [comment]
Fixed in **[v0.22.2](https://github.com/SoCo/SoCo/releases/tag/v0.22.2)**.
